runtime/drivers/clickhouse/clickhouse.go

```diff
@@ -324,15 +324,6 @@ func (c *connection) AsModelManager(instanceID string) (drivers.ModelManager, bo
 
 // AsTransporter implements drivers.Connection.
 func (c *connection) AsTransporter(from, to drivers.Handle) (drivers.Transporter, bool) {
-	olap, _ := to.(*connection)
-	if c == to {
-		switch from.Driver() {
-		case "s3":
-			return NewS3Transporter(from, olap, c.logger), true
-		case "https":
-			return NewHTTPTransporter(from, olap, c.logger), true
-		}
-	}
```

Comment:
Do you know if anyone were using these? I think not, but if yes we might need a shim. The long term plan is to emit `type: source` as models from the parser, so we don't have to handle them differently at all, but we can't do that until all the transporters have been migrated.

Reply:
I don't think anybody is using them. Most of the current use cases rely on data being populated offline.
These implementations were pretty bad and had following issues:
1. Credentials have to be set on clickhouse server
2. If credentials are set on clickhouse server we can't use any anonymous sources. 
3. Can not have any column override which is required in mostly all cases.
4. Can not set any sorting keys or primary keys which means clickhouse need to scan entire table for every query,

The current approach handles all these cases plus we can also support all possible integrations. Example model that fetches data from s3 with current implementation.
```
type: model
sql: SELECT * s3(...)

output:
  columns: (id UInt32, username varchar, email varchar, created_at datetime, INDEX idx1 username TYPE set(100) GRANULARITY 3) # also adds an index
  materialize: true
  order_by: (id, created_at)

refresh:
  cron: '* * * * *'
```

The clickhouse transporters were added only for the POC and I believe we should remove them given the shortcomings listed above.

Reply:
Good, let's not worry about this then